packages/angular/src/generators/init/init.ts

```diff
@@ -178,7 +188,10 @@ async function addUnitTestRunner(
   }
 }
 
-function addE2ETestRunner(tree: Tree, options: Schema): GeneratorCallback {
+async function addE2ETestRunner(
+  tree: Tree,
+  options: Schema
+): Promise<GeneratorCallback> {
```

Comment:
This was not returning a `Promise,` and as far as I can tell, it's still not returning a `Promise`.

Reply:
yes. revert it.